inc/admin/ui/notices.php

```diff
@@ -559,15 +611,7 @@ function rocket_thank_you_license() {
 	?>
 		<div class="notice notice-success">
 			<p>
-				<strong><?php echo WP_ROCKET_PLUGIN_NAME; ?></strong>: <?php _e( 'Thank you. Your license has been successfully validated!', 'rocket' ); ?><br />
-				<?php
-				printf(
-					/* translators: %1$s license key; %2$s = email address */
-					__( 'Key: <code>%1$s</code><br>Email: <em>%2$s</em>', 'rocket' ),
-					get_rocket_option( 'consumer_key' ),
-					get_rocket_option( 'consumer_email' )
-				);
-				?>
+				<strong><?php echo WP_ROCKET_PLUGIN_NAME; ?></strong>: <?php _e( 'is good to go!', 'rocket' ); ?>
```

Comment:
<?php
    // translators: %s = plugin name.
    printf( __( '%s: is good to go!', 'rocket' ), '<strong>' . WP_ROCKET_PLUGIN_NAME . '</strong>' );
    ?>